examples/minipng/minipng.rflx

```diff
@@ -32,13 +32,14 @@ package MiniPNG is
          Magic        : Opaque
             then Tag
                if Magic = "Mini-PNG";
-         Tag          : Header_Type;
-         Length       : Header_Length;
+         Tag          : Block_Type
+            then Length if Tag = Header;
+         Length       : Block_Length;
```

Comment:
Shouldn't we check that `Length` is always 9 here? The length of the header block is fixed to 9 byte. Any file that has a different length either has another header which we don't parse or an invalid length field. In both cases the file should be rejected.

Reply:
Yes, restricting the length is a good idea here. Rather than hard-coding the length we could also use the size of the other fields as done in 570b5a5. This is more maintainable should any of the scalar types change in the future.

Reply:
This is a good point. I didn't recognize that the restriction was checked there also.